### ISSUE TITLE
[FEAT] Increase Accessibility and Responsive Design Elements

### ISSUE THREAD

**Issue by ShelbyEliza** [ShelbyEliza/front-end-portfolio#21]
The site should be easily used by people with accessibility needs or on any screen size.
